tests/test_mcnp.py

```diff
@@ -457,14 +457,18 @@ def test_xsdir():
     assert_equal(xsdir.tables[0].entries, 0)
     assert_equal(xsdir.tables[0].temperature, 5.5555E+05)
     assert_false(xsdir.tables[0].ptable)
+    assert_equal(xsdir.tables[0].filename, 'such_data/1001.777nc')
```

Comment:
Shouldn't it be like this?
```suggestion
    assert_equal(xsdir.tables[1].filename, 'such_data/1001.777nc')
```

Reply:
I've got it going through CI via my fork - it just reported that exact error. Fix pushed. Thanks for catching it too!